test/e2e/namespace_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Whitelist
+// This should be
```

Comment:
The comment here, and in scale.go, won't make sense to a later reader without seeing the PR that introduced it. Is the idea to use these to test the check, and drop them from the actual merged PR?

Reply:
The comments will be dropped before merge.